tests/test_code_review_fixes.py

```
@@ -0,0 +1,604 @@
+            self.fail("Division by zero occurred with zero relations")
+
+
+class TestDCMA14ConstraintCheck(unittest.TestCase):
+    """Test fix for duplicate constraint check in analysis.py."""
+
+    def test_constraint_list_has_both_values(self):
+        """Test that constraint list includes both CS_MANDSTART and CS_MANDFIN."""
+        from xerparser.dcma14.analysis import DCMA14
+
+        mock_programme = MagicMock()
+        dcma = DCMA14(mock_programme)
```

Ruff (0.14.13)

Local variable `dcma` is assigned to but never used

Remove assignment to unused variable `dcma`

(F841)